packages/core/src/components/popover-core/popover-core.tsx

```diff
@@ -73,7 +76,7 @@ export class TdsPopoverCore {
     cancelable: false,
     bubbles: false,
   })
-  tdsClose: EventEmitter<{}>;
```

Comment:
Are these events exposed on the `tds-popover`? If so, I think this should be considered breaking since users are probably listening to this event and then performing some action. Like refetching/updating data whenever a Popover closes?

Reply:
Ok, so the `tdsClose` event that we had defined here was not exposed to the users?